Makefile

```diff
@@ -30,3 +30,10 @@ pi-dangerous.img: ubuntu-core-desktop-22-pi-dangerous.model $(EXTRA_SNAPS)
 	tar czSf $@ $<
 
 .PHONY: all
+
+clean:
+	sudo rm -rf img
+	sudo rm -rf image2
+	sudo rm -rf output
+	sudo rm -rf image/isolinux
```

Comment:
Is there a reason we don't just remove the whole image directory? That removes the need to explicitly list the yaml file below.

Reply:
I am working with the "installer_builder" branch, and there are some files inside that... I can change it in "main", if you prefer.

Reply:
Removed.